node/src/node_configurator/node_configurator_standard.rs

```diff
@@ -909,6 +1819,31 @@ mod tests {
         );
         let set_gas_price_params = set_gas_price_params_arc.lock().unwrap();
         assert_eq!(*set_gas_price_params, vec![gas_price]);
+    }
+
+    #[test]
+    fn configure_database_with_data_specified_on_command_line_and_in_database_without_seed() {
```

Comment:
An old test which I believe has lost any value from our current view. Dump it please.

Reply:
Why do you say it has lost value?

Reply:
We discussed it on Zoom; I'm leaving it in for now.